newIDE/app/src/UI/PlaceholderError.js

```diff
@@ -4,27 +4,34 @@ import * as React from 'react';
 import { Column, Line } from './Grid';
 import RaisedButton from './RaisedButton';
 import EmptyMessage from './EmptyMessage';
+import GDevelopThemeContext from './Theme/ThemeContext';
 
 type Props = {|
   children: React.Node,
   onRetry?: () => void | Promise<void>,
+  kind?: 'error' | 'valid' | 'warning',
```

Comment:
I did not think it through, I just copied the possible values in the theme. Maybe it could be useful to display an error, and when it is resolved, it is then displayed in green saying "Error is fixed".

Maybe it's far-fetched and I can remove this option